### ISSUE TITLE
Error message with OEBB

### ISSUE THREAD

**Issue by gaudes** [public-transport/hafas-client#213]
Hi Jannis,

sorry, me again.

A user and I get an error message when querying OEBB:

```
Error: CLIENTVERSION
  File "/opt/iobroker/node_modules/hafas-client/lib/request.js", line 86, col 14, in Object.request
    const err = new Error()
  File "/opt/iobroker/node_modules/hafas-client/index.js", line 282, col 18, in Object.locations
    return profile.request({profile, opt}, userAgent, req)
  File "/opt/iobroker/node_modules/iobroker.fahrplan/lib/helper.js", line 94, col 39, in fHelpers.getStation
    const sResult = await this.hClient.locations(sSearchString, {results: 10, stops: true, addresses: false, poi: false});
  File "/opt/iobroker/node_modules/iobroker.fahrplan/main.js", line 94, col 46, in Fahrplan.onMessage
    const jSearchResult = await this.helper.getStation(obj.message.provider, obj.message.station);
  File "events.js", line 314, col 20, in Fahrplan.emit
  File "domain.js", line 483, col 12, in Fahrplan.EventEmitter.emit
  File "/opt/iobroker/node_modules/iobroker.js-controller/lib/adapter.js", line 5483, col 34, in change
    this.emit('message', obj);
  File "/opt/iobroker/node_modules/@iobroker/db-states-redis/lib/states/statesInRedisClient.js", line 232, col 37, in Immediate.<anonymous>
    onChange(channel.substring(this.namespaceRedisL), message);
  File "internal/timers.js", line 461, col 21, in processImmediate
```

The error occurs since today about 5:30 when querying for stations.

Any ideas ?

Thanks and regards,

Ralf

**Comment by derhuerst** [comment]
I didn't change anything significant at this time. Probably they updated the ÖBB endpoint and raised the minimum protocol version.

I will look into this tomorrow.

**Comment by derhuerst** [comment]
Thanks for reporting!

Fixed in 7025d3b and published as `hafas-client@5.15.1`. 🎉
